charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -138,7 +138,6 @@ spec:
                     type: object
                     required:
                       - secretname
-                      - userkey
```

Comment:
Why does it not required?

Reply:
when we can have a default user either that or the userkey has to be set. That's why I excluded it here

Reply:
Userkey is mandatory, default value should kicks in when a userkey value is not found within a secret, no?

Reply:
true, mixed it up in my head